src/tools/PrizeTierController/SavePrizeTiersModal/index.tsx

```diff
@@ -0,0 +1,132 @@
+}
+
+export const SavePrizeTiersModal = () => {
+  const [isOpen, setIsOpen] = useAtom(isSavePrizeTiersModalOpenAtom)
+  const [modalState, setModalState] = useState<SavePrizeTiersModalState>(
+    SavePrizeTiersModalState.review
+  )
+  const prizePools = usePrizePools()
+  const { data, isFetched } = useAllPrizeTierHistoryData()
+  const [combinedPrizeTiers] = useAtom(allCombinedPrizeTiersAtom)
+
+  const allPrizePoolConfigEdits = useMemo(() => {
+    if (isFetched) {
+      return prizePools.map((prizePool) => {
+        if (combinedPrizeTiers[prizePool.chainId]) {
+          const oldConfig = data[prizePool.chainId][prizePool.prizeTierHistoryMetadata.address]
+          const newConfig =
+            combinedPrizeTiers[prizePool.chainId][prizePool.prizeTierHistoryMetadata.address]
+          const edits = checkForPrizeEdits(oldConfig, newConfig)
+          return { prizePool, oldConfig, newConfig, edits }
+        }
+      })
+    }
+  }, [prizePools, data, isFetched, combinedPrizeTiers])
+
+  return (
+    <BottomSheet
+      isOpen={isOpen}
+      closeModal={() => {
+        setIsOpen(false)
+        setModalState(SavePrizeTiersModalState.review)
+      }}
+    >
+      {modalState === SavePrizeTiersModalState.review && (
+        <ReviewEdits
+          allEdits={allPrizePoolConfigEdits}
+          onContinue={() => setModalState(SavePrizeTiersModalState.txs)}
+        />
+      )}
+      {modalState === SavePrizeTiersModalState.txs && (
+        <SaveEdits allEdits={allPrizePoolConfigEdits} />
+      )}
+    </BottomSheet>
+  )
+}
+
+const ReviewEdits = (props: { allEdits: PrizePoolEditHistory[]; onContinue: Function }) => {
+  const { allEdits, onContinue } = props
+  const [isRawDisplay, setRawDisplay] = useState(false)
+
+  return (
+    <>
+      <p>Review Edits:</p>
+      <button className='mb-4 opacity-60' onClick={() => setRawDisplay(!isRawDisplay)}>
+        {isRawDisplay ? 'Hide' : 'Show'} raw values
+      </button>
+      <ul className='flex flex-col gap-4 mb-4'>
+        {allEdits.map((editHistory) => (
+          <PrizePoolEditsDisplay
+            prizePool={editHistory.prizePool}
+            oldConfig={editHistory.oldConfig}
+            newConfig={editHistory.newConfig}
+            edits={editHistory.edits}
+            displayRawValues={isRawDisplay}
+            key={`prizePoolEdits-${editHistory.prizePool.id()}`}
+          />
+        ))}
+      </ul>
+      {allEdits.every((editHistory) => !editHistory.edits.edited) ? (
+        'No edits.'
+      ) : (
+        <Button type='button' onClick={() => onContinue()}>
+          Continue
+        </Button>
+      )}
+    </>
+  )
+}
+
+const SaveEdits = (props: { allEdits: PrizePoolEditHistory[] }) => {
+  const { allEdits } = props
+  const prizePools = usePrizePools()
+  const { data, isFetched } = usePrizeTierHistoryNewestDrawId(prizePools[0])
+  const [drawId, setDrawId] = useState(0)
+
+  return (
+    <>
+      <p className='mb-4'>Submit Transactions:</p>
```

Comment:
nit: use the `title` prop on the `BottomSheet`

Reply:
Added through d028aca